### ISSUE TITLE
Can a version be made that supports the stock rf module?

### ISSUE THREAD

**Issue by robthomson** [fishpepper/OpenGround#2]
As per subject.. is it necessary to use the flysky module?

For many... the stock module will be perfect - what they would want is an opensource firmware.

**Comment by JackFarrand** [comment]
Agreed on this, ability to add more mixes and wing types would be brilliant.

Perhaps a bounty?

**Comment by robthomson** [comment]
And more model memory!

**Comment by andymac4182** [comment]
@JackFarrand I don't think this is a great radio for planes and mixes due to no trims.

**Comment by JackFarrand** [comment]
@andymac4182 see, you say that but I use mine for three separate fixed-wing models, trims are just done using the back buttons and the sticks, very easy to use.

**Comment by Tesla99** [comment]
Yes an open source firmware on stock module with more mixing options would be best. 
While ability to use frsky module is also great.

**Comment by duststorm** [comment]
I'm going to look into this. As part of getting open source software on my Evolution, I first want to try and get the standard tx module working, and then perhaps see if I can exchange it with a module that supports frsky D16.
The people at deviation already have implemented AFHDS2, so it should be a matter of porting that code to the stm32 in the i6s/evolution.

**Comment by jacobbloy** [comment]
i don't know about others but i think maybe the support for a multi 4 in 1 module connected via serial could work, opentx added support for this a while back and i think we could use the same method. then we can choose the protocol.

**Comment by rav-rav** [comment]
Do you mean this project? https://github.com/pascallanger/DIY-Multiprotocol-TX-Module
That really looks very interesting.

**Comment by jacobbloy** [comment]
Yes this module, but you can also use his firmware on the banggood 4in1 modules. In opentx you can plug this module in via serial and then bind set the protocol all via the model setting.

**Comment by odi-um** [comment]
@duststorm have you made any progress ?
